README.md

```diff
@@ -29,6 +29,19 @@ To deploy the application:
     * For example: `./deploy.sh testing` will deploy to `sf-dahlia-pl-testing.herokuapp.com`
   * This works from any branch, for example if you have checked out the `dev` branch and you `./deploy.sh new-menu` then the dev branch will be deployed to `sf-dahlia-pl-new-menu.herokuapp.com`
 
+## Adding New Icons
```

Comment:
Thank you for this!

Just checking, if we ever have a custom icon from Chris, I'm guessing the process will be totally different, right? I guess we can just keep in mind to update this documentation if that ever happens

Reply:
yeah we'll have to do that for the numbered circles, add them outside of the icomoon folder definitely.